platform/core/src/services/MeasurementService/MeasurementService.ts

```diff
@@ -529,6 +526,7 @@ class MeasurementService extends PubSubService {
       measurement = toMeasurementSchema(sourceAnnotationDetail);
       measurement.source = source;
     } catch (error) {
+      console.log('Failed to map', error);
```

Comment:
give more information 
`MeasurementService::annotation ........`

Reply:
The stack trace has all that information, this is just to be able to debug.